homeassistant/components/cast/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Google Cast",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/cast",
-  "requirements": ["pychromecast==12.1.4"],
+  "requirements": ["pychromecast==13.0.0"],
```

Comment:
The version info seems stale:
https://github.com/home-assistant-libs/pychromecast/blob/ba30afcc0138c4a749a43d430f419d99a5eb7fb9/pychromecast/__init__.py#L31

Reply:
What's the breaking change in 13.0.0?

Reply:
You're right, the version info is not kept up to date.
I think the agreement was to just remove it.

The breaking change in 13.0.0 is that pychromecast no longer pins protobuf < 4.

Reply:
This now bumps to 13.0.1 where the `__version__` is removed